qa/tasks/cephfs/filesystem.py

```diff
@@ -595,7 +595,7 @@ def reset(self):
         self.run_ceph_cmd("fs", "reset", str(self.name), '--yes-i-really-mean-it')
 
     def fail(self):
-        self.run_ceph_cmd("fs", "fail", str(self.name))
+        self.run_ceph_cmd(f'fs fail {self.name} --yes-i-really-mean-it')
```

Comment:
why change to passing a string here?

Reply:
It's more convenient to modify a string compared to list of strings.

Reply:
Reversed.